app/v2/controllers/ErrorTranslator.scala

```diff
@@ -64,6 +64,8 @@ trait ErrorTranslator {
   implicit val persistenceErrorConverter = new Converter[PersistenceError] {
 
     def convert(persistenceError: PersistenceError): PresentationError = persistenceError match {
+      case PersistenceError.DepartureNotFound(movement) =>
```

Comment:
Not a big deal, but we might want to be consistent with naming, mostly we refer to this as a departureid.

Reply:
You're right - I should make this consistent, so I've done so!

I'm less bothered about the one below, as a note, as that might end up with a quasi union type (because it might end up applying for arrivals too...)